@here/harp-mapview/lib/text/TextElement.ts

```diff
@@ -327,6 +327,12 @@ export class TextElement {
 
     pathLengthSqr?: number;
 
+    /**
+     * Time to fade in text in milliseconds.
+     * @default [[DEFAULT_FADE_TIME]] 800
+     */
+    textFadeTime?: number;
+
```

Comment:
Out of the scope of this change, but we should take all properties related to how a text element is rendered into a separate style object, much like TextRenderStyle & TextLayoutStyle.